app/controllers/auth0_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class Auth0Controller < ApplicationController
+  skip_before_action :authenticate_user!, :verify_authenticity_token
+
+  include Auth0Helper
+
+  def auth0_callback
+    # Store the user token that came from Auth0 and the IdP
+    # Auth0Helper#auth0_session
+    self.auth0_session = request.env['omniauth.auth'].credentials.id_token
+
+    # Redirect to the URL you want after successful auth
+    redirect_to "/home"
```

Comment:
can you do `redirect_to root_path` instead

Reply:
I noticed `root_path == "/"`. I wanted to redirect to home, since user is already authenticated. Let me see if there is a more elegant way to do that.

Reply:
Ok, looks like `home_path` is what I was looking for